docs/data/charts/pie-demo/OnSeriesItemClick.js

```diff
@@ -32,30 +33,37 @@ export default function OnSeriesItemClick() {
     <Stack
       direction={{ xs: 'column', md: 'row' }}
       alignItems={{ xs: 'flex-start', md: 'center' }}
+      gap={2}
       justifyContent="space-between"
-      sx={{ width: '100%' }}
+      sx={{ width: '100%', overflow: 'hidden' }}
     >
       <Typography
         component="pre"
         sx={{ maxWidth: { xs: '100%', md: '50%', flexShrink: 1 }, overflow: 'auto' }}
       >
-        {`item id: ${id ?? 'undefined'}
-
-item identifier:
-${formatObject(identifier)}`}
+        item id:
+        <br />
+        {id ?? 'undefined'}
+        <br />
+        <br />
+        item identifier:
+        <br />
+        {formatObject(identifier)}
       </Typography>
```

Comment:
I'm wondering if [this demo](https://deploy-preview-15298--material-ui-x.netlify.app/x/react-charts/pie-demo/#onseriesitemclick) should not just be deleted since it's mostly a duplicate of [this one](https://deploy-preview-15298--material-ui-x.netlify.app/x/react-charts/pie/#click-event)

Reply:
def